spec/controllers/api/groups_controller_spec.rb

```diff
@@ -12,6 +12,11 @@
     sign_in user
   end
 
+  describe 'export' do
+    it 'kicks off a group export job'
```

Comment:
Would be good to write these, even if it's just a check for 200 and 403. Maybe check to ensure Document.count is incremented too.